src/interaction/TrackUpdater.ts

```diff
@@ -73,10 +73,11 @@ export default class TrackUpdater {
       if (after) {
         geometryUpdates.push(this.router.snapSegment(after, modifiedControlPoint, pointTo));
       }
+      const results = await Promise.all(geometryUpdates);
```

Comment:
This is done first so that it can throw without side effect?
Can you add a comment?

Reply:
we should wait until snapping is done before equalize coordinates, I added a comment